vllm/model_executor/models/molmo.py

```diff
@@ -1121,9 +1121,26 @@ def input_processor_for_molmo(ctx: InputContext, inputs: DecoderOnlyInputs):
 @MULTIMODAL_REGISTRY.register_max_image_tokens(get_max_molmo_image_tokens)
 @INPUT_REGISTRY.register_dummy_data(dummy_data_for_molmo)
 @INPUT_REGISTRY.register_input_processor(input_processor_for_molmo)
-class MolmoForCausalLM(nn.Module, SupportsMultiModal, SupportsPP):
-
-    def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
+class MolmoForCausalLM(nn.Module, SupportsMultiModal, SupportsPP,
+                       SupportsLoRA):
+    packed_modules_mapping = {
+        "att_proj": ["att_proj"],
+        "attn_out": ["attn_out"],
+        "ff_proj": ["ff_proj"],
+        "ff_out": ["ff_out"],
+    }
+    supported_lora_modules = [
+        "att_proj",
+        "ff_proj",
+    ]
+    embedding_modules = {}
+    embedding_padding_modules = {}
+
+    def __init__(self,
+                 *,
+                 vllm_config: VllmConfig,
+                 prefix: str = "",
+                 lora_config: Optional[LoRAConfig] = None):
```

Comment:
We can obtain `lora_config` directly from `vllm_config` instead of adding it as a separate argument

Reply:
fixed that